Lib/multiprocessing/shared_memory.py

```diff
@@ -8,34 +8,120 @@
 
 from functools import reduce
 import mmap
-from .managers import DictProxy, SyncManager, Server
+from .managers import dispatch, BaseManager, Server, State, ProcessError, \
```

Comment:
What do you think about use `()` on imports? This way you avoid use `\`

Reply:
I like both styles.  At the moment, I think it best to stay consistent with the pattern that already exists in multiprocessing and its submodules.